File: MODULO 01/SEMANA 05/Exercicios S05/Ex04 - Módulos Export e Import/modules/Pessoa.js

export default class Pessoa {
    #_cpf
    constructor(nome, cpf){
        this.nome = nome 
        this.#cpf = cpf
    }

    set #cpf(cpf){
        this.#_cpf = cpf
    }

    get #cpf(){
        return this.#_cpf
    }

    imprime(){
        return `Nome: ${this.nome} CPF: ${this.#cpf}`
    }

}

File: MODULO 02/SEMANA 05/Exercicios M2S05/pessoa.py
import json
from exception import AtributoException
from Endereco import Endereco


class Pessoa:
    def __init__(self) -> None:
        self.nome = None
        self.celular = None
        self.email = None
        self.endereco = Endereco()

    def cadastrar_pessoa(self, nome: str, celular: str, email: str):
        pass

    def exibir_pessoa(self):
        pass

    def __salvar_pessoa(self):
        pass
